AltServer/AppDelegate.swift

```diff
@@ -222,6 +195,15 @@ extension AppDelegate: NSMenuDelegate
         
         return true
     }
+    
+    func menuWillOpen(_ menu: NSMenu)
+    {
+        if menu == self.appMenu
+        {
+            self.connectedDevices = ALTDeviceManager.shared.connectedDevices
+            self.launchAtLoginMenuItem.state = LaunchAtLogin.isEnabled ? .on : .off
```

Comment:
We can remove self prefixes here. 

if menu == appMenu {
    connectedDevices = ALTDeviceManager.shared.connectedDevices